### ISSUE TITLE
Find the best way to accommodate customisations

### ISSUE THREAD

**Issue by hughbris** [hughbris/grav-plugin-admin-addon-toast-ui-editor#2]
I have successfully hardcoded a custom button with popup that mostly does what I need it to. I did have in mind to build this (mostly) JS from values in the plugin config, but I think now that will be an arduous task for very little benefit. I am prepared to expect that administrators of the plugin will be reasonably confident with some hacking.

I thought I could supply some customisations as a sample.

I have moved the inline script to a partial template for now in the admin and `include`d it. It needs to be a Twig file to access some Grav page values. I thought if I just scan a directory for files following a filename pattern, then administrators can just drop their customisation partials in there.

In summary and in fancy words, I'm searching for an elegant extensibility framework.
